gcp/README.md

```diff
@@ -555,38 +555,32 @@ In this scenario we rely on CouchDB ability to recover from loss of one or more
    * You can check DB status on recovered node with `kubectl exec --namespace gpii -it couchdb-couchdb-N -c couchdb -- curl -s http://$TF_VAR_couchdb_admin_username:$TF_VAR_couchdb_admin_password@127.0.0.1:5984/gpii/`, where N is node index.
 1. To make sure that all systems are functional, run smoke tests with `rake test_preferences_read` and then `rake test_flowmanager`.
 
-### Data corruption on all replicas of CouchDB cluster
+### Restoring Entire CouchDB Cluster from Snapshots
 
-There may be a situation, when we want to roll back entire DB data set to another point in the past. Current solution is disruptive, requires bringing entire CouchDB cluster down and some manual actions (we'll most likely automate this in future).
+There may be a situation, when we want to roll back entire DB data set to
+another point in the past. This operation is disruptive and requires bringing
+entire CouchDB cluster down.
 
-Ops team must perform backup restoration test using this scenario on `gpii-gcp-stg` cluster monthly, to make sure that:
-* Automated backups are being created as expected.
-* Existing backups can be used to restore functional and consistent DB.
-* Restoration guide (this scenario) is accurate.
```

Comment:
One of the purposes of the restoration test is to keep the guide accurate, so I think this should be put into rephrased version as well.

Reply:
Done.